Meta/CMake/time_zone_data.cmake

```diff
@@ -40,16 +40,8 @@ set(TZDB_ZONE_1970_PATH "${TZDB_PATH}/${TZDB_ZONE_1970_SOURCE}")
 
 function(extract_tzdb_file source path)
     if(EXISTS "${TZDB_ZIP_PATH}" AND NOT EXISTS "${path}")
-        if (CMAKE_VERSION VERSION_LESS 3.18.0)
-            message(STATUS "Extracting using ${TAR_TOOL} file ${source}")
-            execute_process(COMMAND "${TAR_TOOL}" -C "${TZDB_PATH}" -xzf "${TZDB_ZIP_PATH}" "${source}" RESULT_VARIABLE tar_result)
-            if (NOT tar_result EQUAL 0)
-                message(FATAL_ERROR "Failed to unzip ${source} from ${TZDB_ZIP_PATH} with status ${tar_result}")
-            endif()
-        else()
-            message(STATUS "Extracting using cmake ${source}")
-            file(ARCHIVE_EXTRACT INPUT "${TZDB_ZIP_PATH}" DESTINATION "${TZDB_PATH}" PATTERNS "${source}")
-        endif()
+        message(STATUS "Extracting using cmake ${source}")
```

Comment:
I think the original author added the "using cmake" part of the text to help differentiate what extractor is used when people came into #build-problems - I'd actually be in favor of removing this message now (here and in other .cmake files) to reduce build noise

Reply:
Sure. Doesn't have to be in this PR though maybe? Seems unrelated to cmake version stuff.

Reply:
Sure